frontend/src/game/Grid.jsx

```diff
@@ -161,7 +161,7 @@ CardImage.propTypes = {
   loyalty:  PropTypes.string,
   setCode: PropTypes.string,
   number: PropTypes.string,
-  scryfallId: PropTypes.string,
+  identifiers: PropTypes.string,
```

Comment:
identifiers is an Array of strings, not a string

Reply:
I need to change this, I forgot about this one.

Reply:
at the end, I didn't use so I will remove it

Reply:
changed